sonata-project/core-bundle/3.7/config/packages/sonata_project/sonata_core.yaml

```diff
@@ -0,0 +1,7 @@
+sonata_core:
+    flashmessage: {}
+    form:
+        mapping:
+            enabled: false
+framework:     # Remove this section
+    session: ~ # if you'll be using this in config/packages/framework.yaml
```

Comment:
@aLkRicha session is enabled by default now, so you can remove `framework` configuration. See https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/config/packages/framework.yaml